### PR TITLE
Quick fix for #7127

### PR DESCRIPTION
**Pull request by robertpustulka**

### CODE REVIEW

File: src/ORM/Behavior/TranslateBehavior.php [hunk 1]
```diff
@@ -439,24 +439,26 @@ protected function _rowMapper($results, $locale)
...
-            foreach ($grouped->combine('field', 'content', 'locale') as $locale => $keys) {
-                $entityClass = $this->_table->entityClass();
-                $translation = new $entityClass($keys + ['locale' => $locale], [
-                    'markNew' => false,
-                    'useSetters' => false,
-                    'markClean' => true
-                ]);
-                $result[$locale] = $translation;
-            }
+            if ($row instanceof EntityInterface) {
```
Comment (lorenzo) [inline src/ORM/Behavior/TranslateBehavior.php]: would you mind writing it in the opposite way?

```php
if (!$row instanceof EntityInterface) {
    return $row;
}
...
```

**Comment by robertpustulka** [comment]
@lorenzo done

**Comment by lorenzo** [comment]
Thanks!

**Comment by lorenzo** [comment]
@robertpustulka Are you in IRC by change?

**Comment by robertpustulka** [comment]
@lorenzo, no, actually I've never used IRC :) why are you asking?

**Comment by lorenzo** [comment]
@robertpustulka Would you mind sending me an email? It is in my github profile, I'd like to share an idea

**Pull request merged**